src/sentry/static/sentry/app/views/settings/account/accountSecurity/components/removeConfirm.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import {t} from '../../../../../locale';
+import ConfirmHeader from './confirmHeader';
+import Confirm from '../../../../../components/confirm';
+import TextBlock from '../../../components/text/textBlock';
+
+class RemoveConfirm extends React.Component {
+  render() {
+    return (
+      <Confirm
+        message={
+          <React.Fragment>
+            <ConfirmHeader>{t('Do you want to remove the method?')}</ConfirmHeader>
+            <TextBlock>
+              {t(
+                'You will no longer be able to use it for two-factor authentication afterwards. Removing the last authenticator removes two-factor authentication completely.'
```

Comment:
What does "the last authenticator" mean? From what I could tell you can only have one

Reply:
You can have multiple methods (e.g. SMS + u2f). I agree, the msg is kind of awkward

Reply:
I think if we just make this "Removing the last authentication method....." it would already be quite a bit clearer

Reply:
"Authenticator" is such a weird word

Reply:
Doesn't it seem kind of redundant?